### ISSUE TITLE
Local development: CSS changes disappear when updating non-css files

### ISSUE THREAD

**Issue by edbrannin** [surjithctly/neat-starter#28]
Duplicate of already-closed-without-comment #23 

## Steps to reproduce:

0. Make some obvious changes to `tailwind.css`
1. Run `npm start`
2. Save `index.md`
3. Add a blank line to the end of `tailwind.css` and save

## Expected result:

My changes to tailwind.css are always active

## Observed result:

My changes stop working after step 2, but work again after step 3.

## Notes

Touching the CSS every time I update any other page on the site is... not the most pleasant workflow. :)

I've noticed the Chrome inspector says the CSS file is `styles.css` when I've touched any non-CSS content, and `style.css?browsersync=timestamp` when I've touched the CSS.

**Comment by kxzo** [comment]
Same, here!
